x/consensus/depinject.go

```diff
@@ -72,3 +74,37 @@ func ProvideModule(in ModuleInputs) ModuleOutputs {
 		BaseAppOption: baseappOpt,
 	}
 }
+
+type versionModifier struct {
```

Comment:
In theory, we could remove it from baseapp and use this one too in app v1 right?

Reply:
If so, let's create a constructor to use in not depinject app.go.
For depinject app.go, removing the provider from runtime v1 should be sufficient I think.